packer/README.md

```diff
@@ -21,19 +22,27 @@ enable:
 
     git clone https://github.com/rackhd/rackhd
     cd rackhd/packer
-    packer build -only=virtualbox-iso template-ubuntu-14.04.json
-
+    packer build -only=virtualbox-iso templates/template-ubuntu-14.04-source.json
+    
 ### Build vagrant box based on Ubuntu 16.04
 
     git clone https://github.com/rackhd/rackhd
     cd rackhd/packer
-    packer build -only=virtualbox-iso template-ubuntu-16.04.json
-
+    packer build -only=virtualbox-iso templates/template-ubuntu-16.04-source.json
+    
 ### To build locally (using pre-built debian packages - Ubuntu 14.04)
 
     git clone https://github.com/rackhd/rackhd
     cd rackhd/packer
-    packer build -only=virtualbox-iso template-packages.json
+    packer build -only=virtualbox-iso templates/template-ubuntu-14.04-packages.json
+
+
+### To build locally without publishing to atlas, use the `-notlas` variant.
```

Comment:
[non-issue]
the -notlas.json is quite overlay with original .json.
is it possible we introduce someway to turn on/off the Atlas publishing , with single .json ?
though I didn't find it , seems only "only" tag can be used but not apply the this de-duplication need...